runtime/vm/jniinv.c

```diff
@@ -869,7 +869,18 @@ jint JNICALL GetEnv(JavaVM *jvm, void **penv, jint version)
 #endif /* defined(J9VM_OPT_JAVA_OFFLOAD_SUPPORT) */
 
    if (!jniVersionIsValid((UDATA)version)) {
-		return JNI_EVERSION;
+#if JAVA_SPEC_VERSION >= 11
```

Comment:
This code is unncessary and incorrect. This file does not require any changes at all.

Reply:
This is actually the origination of this PR, otherwise `test/hotspot/jtreg/serviceability/AsyncGetCallTrace/MyPackage/ASGCTBaseTest.java` failed with following message.
```
Error: wrong result of a valid call to GetEnv!
JVMJ9TI064E Agent initialization function Agent_OnLoad failed for library AsyncGetCallTraceTest, return code -1
JVMJ9VM015W Initialization error for library j9jvmti29(-3): JVMJ9VM009E J9VMDllMain failed
```
It passes w/ this PR.

Reply:
It will pass without this code as well. GetEnv doesn't deal with JVMTI here - it calls a hook to validate the version.

Reply:
Updated PR to remove it from `GetEnv()`.

Please have another look.